openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -49,4 +47,4 @@ def handle(self, *args, **options):
             if not course_keys:
                 log.fatal('No courses specified.')
```

Comment:
@dsjen Do you recall why these are `log.fatal` calls instead of `raise CommandError` calls?  If there's no good reason, I'll change them to `raise CommandError` calls instead.